### PR TITLE
Implemented the account and the following classes. Started the implem…

### PR DESCRIPTION
**Pull request by ptomo**
…entation of the persistence interface.

### CODE REVIEW

File: src/main/java/com/byop/instamine/model/Account.java [hunk 1]
```diff
@@ -44,4 +49,24 @@ public String getEmailAddress() {
...
+
+    public List<String> getFollowings() {
+        return followings;
+    }
+
+    public void addFollowing(String following) {
+        if(!followings.contains(following))
+            followings.add(following);
+        else
+            throw new IllegalArgumentException("Follower: " + following + " already exists in the list for account: " + toString());
```
Comment (broy94) [inline src/main/java/com/byop/instamine/model/Account.java]: Could introduce own type of Exception class - eg.: InstaMineException. Let's wrap all exceptions in this class later on. (Please implement in next commit.)

File: src/main/java/com/byop/instamine/persistence/FilePersistence.java [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+            if(file.exists())
+                file.delete();
+
+            file.mkdirs();
+            FileOutputStream outputStream = new FileOutputStream(file, true);
+            OutputStreamWriter outputStreamWriter = new OutputStreamWriter(outputStream, charset);
+            BufferedWriter bufferedWriter = new BufferedWriter(outputStreamWriter);
+            writer = new PrintWriter(bufferedWriter, true);
+        } catch (FileNotFoundException f) {
+            System.out.println("File not found: " + filePath + " error message: " + f.getMessage());
```
Comment (broy94) [inline src/main/java/com/byop/instamine/persistence/FilePersistence.java]: Could use some kind of logger. (Please implement in one of the next commits.)

Comment (ptomo) [reply src/main/java/com/byop/instamine/persistence/FilePersistence.java]: Yes, that's the plan

File: src/main/java/com/byop/instamine/persistence/file/FileLoad.java [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+                    } else {
+                        isReadingFollowings = false;
+                    }
+                }
+
+                if(isReadingFollowings) {
+                    if(line != null) {
+                        String[] splitLine = line.split("->");
+                        if(splitLine.length != 2)
+                            throw new UnsupportedOperationException("Invalid line in file " + filePath + " => " + line);
```
Comment (broy94) [inline src/main/java/com/byop/instamine/persistence/file/FileLoad.java]: Let's use { } brackets in these cases as well

**Pull request merged**